src/main/java/com/ldtteam/blockui/views/ScrollingList.java

```diff
@@ -131,6 +133,17 @@ public interface DataProvider
          */
         int getElementCount();
 
+        /**
+         * Override this to pick a custom size for this element. Event contains the logic to modify the old size.
+         *
+         * @param index the index of the row/list element.
+         * @param event the event used to modify the size.
+         */
+        default void getElementSize(int index, final EventMutableSizeI event)
+        {
```

Comment:
a "get" with void. I dont think that is a good idea

Reply:
modifyElementSize would indeed make more sense :+1:

Reply:
Id rather make this return a decent default, and allow other elements to override the default then if necessary.

Reply:
Unless it actually alters something. But its not super clear how this would look in practice.

Reply:
Its supposed to either be noop, or alter the row size (thus mby modifyRowSize?) if you need to change it anyhow (usually list object type..

Reply:
Okay, then modifyRowSize is fine. But why through an event?

Reply:
> Okay, then modifyRowSize is fine. But why through an event?

Night came up with the idea, with a class we had the option to add methods to modify the size as opposed to letting the user do the math.